libs/utils/analysis/tasks_analysis.py

```diff
@@ -57,15 +57,15 @@ def _dfg_top_big_tasks(self, min_samples=100, min_utilization=None):
             default: capacity of a little cluster
         :type min_utilization: int
         """
-        if not self._trace.hasEvents('sched_load_avg_task'):
-            self._log.warning('Events [sched_load_avg_task] not found')
+        if self._dfg_task_load_events() is None:
```

Comment:
This is kind-of confusing: we check here (L60) and get the df after (L68). That's error prone if we change the code.

I would prefer something like:

```python
df = self._dfg_task_load_events()
if df is None:
    self._log.warning('No trace events for task signals, plot DISABLED')
    return None

# So that from now on we don't use `_dfg_task_load_events()` in this function.
```